src/styles/components/_definition.scss

```diff
@@ -5,34 +5,48 @@ $breakpoint-background-image-size: 75rem;
   width: 100%;
   position: relative;
   margin: 2.5rem 0 0; // top: 40px; left&right: 0; bottom: 0;
+  padding: 8rem 0;
 
-  @media (min-width: $breakpoint-scale-up) {
-    margin-bottom: -8.5625rem; // compensates for code example padding
+  &::after {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: calc(-50vw + 49.5%); // Forbidden Zastrow Knowledge
```

Comment:
Forbidden Zastrow Knowledge 💎 😂

Reply:
@jkohrt7 Please keep this

Reply:
Absolutely love the way this works out.

<img width="149" alt="Screenshot 2022-12-14 at 6 28 34 PM" src="https://user-images.githubusercontent.com/53844657/207758206-41a234e5-5b60-4ba2-a69e-71d33232e2ba.png">